service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43ChildArrangementOrderDocumentParameterGenerator.java

```diff
@@ -100,22 +105,29 @@ public DocmosisTemplates template() {
     private String buildOrderDetails(ManageOrdersEventData eventData) {
         StringBuilder stringBuilder = new StringBuilder();
 
-        stringBuilder.append("The Court orders");
+        stringBuilder.append("The Court orders,");
 
         if (isChildArrangementOrderSelected(eventData)) {
-            stringBuilder.append("\n\nThe Child Arrangement Order is for the child to ");
-            stringBuilder.append(eventData.getManageOrdersChildArrangementsOrderTypes().stream()
-                .map(type -> {
-                    switch (type) {
-                        case CHILD_LIVE:
-                            return "live with";
-                        case CHILD_CONTACT:
-                            return "have contact with";
-                        default: return type.toString();
-                    }
-                })
-                .collect(Collectors.joining(" and ")));
-            stringBuilder.append(".");
+            stringBuilder.append("\n\nThe Child Arrangement Order is for the:");
+            if (eventData.getManageOrdersChildArrangementsOrderTypes().contains(CHILD_LIVE)) {
+                stringBuilder.append("\n\nChild to live with\n\n"
+                    + eventData.getManageOrdersChildArrangementsLiveWithDetails());
```

Comment:
Could we use append rather than concatenating inside the append?

e.g. stringBuilder.append("\n\nChild to live with\n\n")
                         .append(eventData.getManageOrdersChildArrangementsLiveWithDetails());